docs/resources/credits_template.rst

```diff
@@ -0,0 +1,77 @@
+.. _tardiscredits:
+
+******************************
+Credits & Publication Policies
+******************************
+
+|DOI_BADGE|
+
+We provide TARDIS as a free, open-source tool. If you are using it, please
+adhere to a few policies and acknowledge the TARDIS Team.
+
+Publication Policies
```

Comment:
This whole section needs revision to be relevant for Carsus. @wkerzendorf @afloers